LayoutTests/TestExpectations

```diff
@@ -6219,6 +6219,7 @@ imported/w3c/web-platform-tests/import-maps/ [ DumpJSConsoleLogInStdErr ]
 fullscreen/fullscreen-enter-bottom-padding-animation.html [ Skip ]
 
 # Extra logging needed to be silenced:
+fast/dom/set-attribute-and-normalize-in-event.html [ DumpJSConsoleLogInStdErr ]
```

Comment:
Can we edit the test so it displays "This test passes if it does not crash"? That will enshrine the investigation here into the test in perpetuity and we won't have to deal with it again.